src/include/parser/statement.h

```diff
@@ -40,6 +42,7 @@ class Statement {
     }
 
 private:
+    double parsingTime;
```

Comment:
Up to u but I prefer to have `std::vector<std::shared_ptr<Statement>> ClientContext::parseQuery` this function return a struct including Statement & parsing time